vic/drivers/shared_image/src/vic_init.c

```diff
@@ -1106,7 +1107,9 @@ vic_init(void)
         }
         // handle the bare soil portion of the tile
         vidx = veg_con_map[i].vidx[options.NVEGTYPES - 1];
-        Cv_sum[i] += veg_con[i][vidx].Cv;
+        if (vidx != NODATA_VEG) {
+            Cv_sum[i] += veg_con[i][vidx].Cv;
+        }
```

Comment:
So, it's wrong because it assumes that the final veg class in the veg_lib is bare soil - bad assumption, since a given veg_lib might have bare soil in a different position.  Veg_lib files are not all going to be those for which tonic added a bare soil class at the end.

Since the block is simply processing root zones, you could change the logic to always loop over all veg classes, and instead check for root depths > 0 and/or root zones not `MISSING` or `NODATA`.  We should sync up tonic with this.  We should decide: for bare soil, how should the root zone information be represented - seems like `NODATA` would be the best choice since `MISSING